pkgs/by-name/mi/min-ed-launcher/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildDotnetModule,
+  fetchFromGitHub,
+  git,
+}:
+buildDotnetModule rec {
+  pname = "min-ed-launcher";
+  version = "0.11.2";
+
+  src = fetchFromGitHub {
+    owner = "rfvgyhn";
+    repo = "min-ed-launcher";
+    tag = "v${version}";
+    hash = "sha256-p970rsDyqXBbxo18q/ocNRkBKBq2CwJA3tAfDicrTOE=";
+
+    leaveDotGit = true; # During build the current commit is appended to the version
```

Comment:
informational: some git sources are not very reproducible with this option set. Make sure to monitor build failures for this package once it's merged. If it fails to reproduce, you might have to patch out the build feature depending on this.

Reply:
Thx for the info, I'll look into monitoring it.